rust/README.md

```diff
@@ -0,0 +1,151 @@
+# c-of-time Rust subsystem
+
+An environment for hooking and linking to Pokémon Mystery Dungeon: Explorers of Sky: Rust subsystem.
+
+The Rust subsystem extends c-of-time with the ability to compile and link Rust code into
+the game.
+
+Everything you need to get started is in this repository.
+
+## Project setup
+1. Install a Rust nightly toolchain, preferably with [Rustup](https://rustup.rs/).
+2. Install [Python](https://www.python.org/downloads/) and [devkitpro](https://devkitpro.org/wiki/Getting_Started).
```

Comment:
I also had to add `/opt/devkitpro/devkitARM/bin` to my `PATH` because the setup depends on a few binaries in there.

Reply:
Right, forgot to add that, thanks!